api/app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Exceptions;
+
+use Illuminate\Support\MessageBag;
+use Illuminate\Http\Exception\HttpResponseException;
+
+class ValidationException extends HttpResponseException
```

Comment:
The `ValidationException` should probably implemenent `Symfony\Component\HttpKernel\Exception\HttpExceptionInterface` as the error handler (spira/api/app/Http/Controllers/BaseController.php:renderException) method checks if `$e instanceof HttpExceptionInterface` and sets the response code with `$e->getStatusCode()`. We are using the `Symfony\Component\HttpKernel\Exception` elsewhere so we should probably implement their interface for consistency